multisense_description/urdf/multisenseS21B/multisenseS21b.urdf.xacro

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:macro name="multisenseS21b" params="name">
```

Comment:
I'm pretty sure the file name all needs to be done with a capital B to match the camera type as defined in the launch file. Line 19 of the launch file generates the urdf file name from the sensor type parameter, so one or the other needs to be modified